src/components/OrgSettings/ActionItemCategories/CategoryModal.tsx

```diff
@@ -171,6 +171,16 @@ const CategoryModal: FC<InterfaceActionItemCategoryModal> = ({
                 setFormState({ ...formState, name: e.target.value })
               }
               required
+              sx={{
+                '& .MuiOutlinedInput-root': {
+                  '&:hover fieldset': {
+                    borderColor: '#a8c7fa',
```

Comment:
1. Don't use hard coded colors
1. Make this file use CSS exclusively from `src/style/app.module.css`, with the exception of package based CSS.
1. Delete the CSS file it currently uses.
1. Use no CSS in `assets/css/app.css`
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .
1. Use a variable name that describes the function of the color and not the color itself. For example `--search-button-bg` is much better than `--light-blue` 

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296